terraform/do/droplet.tf

```diff
@@ -21,7 +26,7 @@ resource "digitalocean_firewall" "wireguard" {
   inbound_rule {
     protocol         = "udp"
     port_range       = "51820"
-    source_addresses = [local.myip]
+    source_addresses = toset(values(local.client_ips))
```

Comment:
this could just be wide open

Reply:
Yeah it should be wide open. We won't know the IP of all clients.